opensrp-reveal/src/test/java/org/smartregister/reveal/util/RevealJsonFormUtilsTest.java

```diff
@@ -229,6 +229,21 @@ public void testGetRefAppBednetDistributionForm() {
         assertEquals(JsonForm.REFAPP_BEDNET_DISTRIBUTION_FORM, actualFormName);
     }
 
+    @Test
+    public void testGetThailandCaseConfirmationForm() {
+        Whitebox.setInternalState(BuildConfig.class, BuildConfig.BUILD_COUNTRY, Country.THAILAND);
+        String actualFormName = revealJsonFormUtils.getFormName(Constants.EventType.CASE_CONFIRMATION_EVENT, CASE_CONFIRMATION);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Constants.EventType.CASE_CONFIRMATION_EVENT' due to existing static import 'org.smartregister.reveal.util.Constants.EventType.CASE_CONFIRMATION_EVENT'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5622582)